src/cplscheme/tests/ParallelImplicitCouplingSchemeTest.cpp

```diff
@@ -115,7 +115,7 @@ BOOST_AUTO_TEST_CASE(testInitializeData)
   // Create the coupling scheme object
   ParallelCouplingScheme cplScheme(
       maxTime, maxTimeWindows, timeWindowSize, 16, nameParticipant0, nameParticipant1,
-      context.name, m2n, constants::FIXED_TIME_WINDOW_SIZE, BaseCouplingScheme::Implicit, 100);
+      context.name, m2n, constants::FIXED_TIME_WINDOW_SIZE, BaseCouplingScheme::Implicit, 1, 3);
```

Comment:
I generally prefer the following, because it's easier to read and to find-and-replace without introducing errors:

```
  const int minIterations = 1;
  const int maxIterations= 3;
  ...
  ParallelCouplingScheme cplScheme(
      maxTime, maxTimeWindows, timeWindowSize, 16, nameParticipant0, nameParticipant1,
      context.name, m2n, constants::FIXED_TIME_WINDOW_SIZE, BaseCouplingScheme::Implicit, minIterations, maxIterations);
```

But probably arguable.